packages/components/src/components/navigation-item/navigation-item.ts

```diff
@@ -0,0 +1,297 @@
+import '../spinner/spinner';
+import { css } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+import { HasSlotController } from '../../internal/slot';
+import { html, literal } from 'lit/static-html.js';
+import { ifDefined } from 'lit/directives/if-defined.js';
+import componentStyles from '../../styles/component.styles';
+import cx from 'classix';
+import SolidElement from '../../internal/solid-element';
+
+/**
+ * @summary Flexible button / link component that can be used to quickly build navigations. Takes one of 3 forms: link (overrides all other if 'href' is provided), button (default), or accordion (if 'children' slot present).
+ * @status experimental
+ * @since 1.0
```

Comment:
pls change to latest version +1 right before merging.

Reply:
current version is 1.12.0 so 1.12.0 + 1 = 2.12.0?  Is that correct?

Reply:
as this is a feature it only bumps the `minor` version => 1.13.0 (depending on the time you merge).